src/Server.php

```diff
@@ -90,25 +90,35 @@ public function startBrokerSession()
 
         $this->brokerId = $this->validateBrokerSessionId($sid);
     }
-		
-	/**
-	 * Get session ID from header Authorization or from $_GET/$_POST
-	 */
-	protected function getBrokerSessionID(){
-		$headers = getallheaders();
-		
-		if (isset($headers['Authorization'])){
-			return $headers['Authorization'];
-		}
-		if (isset($_GET['sso_session'])) {
-			return $_GET['sso_session'];
-		}
-		if (isset($_POST['sso_session'])) {
-			return $_POST['sso_session'];
-		}
-		
-		return FALSE;
-	}
+
+            /**
+         * Get session ID from header Authorization or from $_GET/$_POST
+         */
+        protected function getBrokerSessionID()
+        {
+            $headers = getallheaders();
+
+            if (isset($headers['Authorization'])){
+               if (strpos($headers['Authorization'], 'Bearer') === 0){
```

Comment:
Do not accept `Authorization` if it doesn't start with `Bearer ` (mind the space). You can use 1 `if`.
_Site note: Please put a space between `){`._